### PR TITLE
Linked home page to freelance page

### PR DESCRIPTION
**Pull request by amyyalex**

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -14,9 +14,14 @@
 .app_navbar-logo{
     flex: 1;
     margin-left: 50px;
-    font-size: 23px;
 }
 
+.graft-logo{
```
Comment (HassanBahati) [inline src/components/Navbar/Navbar.css]: @amyyalex could we standardize the namings...to qraft-logo than qraft-logo

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -13,7 +17,7 @@ const [toggleMenu, setToggleMenu] = useState(false);
   return (
     <nav className='app_navbar'>
         <div className='app_navbar-logo'>
-            <h1>Qraft <br/> <span className='store'>Store</span></h1>
+           <img src={images.logo} alt='homepage' className='graft-logo' />
```
Comment (HassanBahati) [inline src/components/Navbar/Navbar.jsx]: Please lets use qraft-logo than graft-logo for that className

File: src/components/Navbar/Navbar.css (hunk 1 above)
Comment (amyyalex) [reply src/components/Navbar/Navbar.css]: That's a minor mistake but thanks for calling my attention to it

Comment (amyyalex) [reply src/components/Navbar/Navbar.css]: @HassanBahati I have corrected the misspelling. Kindly go ahead with the review and merge
Thanks

**Review by HassanBahati — COMMENTED** [review]
LGTM

**Review by HassanBahati — APPROVED** [review]

**Pull request merged**